src/AccessibilityInsights/MainWindowHelpers/StateMachine.cs

```diff
@@ -153,6 +153,7 @@ public void HandleBackToSelectingState()
             /// make sure that all Mode UIs are clean since new selection will be done. 
             CleanUpAllModeUIs();
 
+#pragma warning disable CA1508 // Dead code warning doesn't apply here
```

Comment:
This is claiming that these 2 lines can never be satisfied, but I don't believe it:
                || (this.CurrentPage == AppPage.Events)
                || (this.CurrentPage == AppPage.Inspect && SelectAction.GetDefaultInstance().IsPaused)